### ISSUE TITLE
Upload MP4 Window Freezes

### ISSUE THREAD

**Issue by angelawong** [YouPHPTube/YouPHPTube#1835]
When I click on Upload a MP4 File, the following window shows up. It does not stop processing and the window freezes. Nothing is clickable on the window, not even to close the window. Please advise. Thank you. 

Version 7.2

![image](https://user-images.githubusercontent.com/1007028/60694347-d6b08680-9eab-11e9-863d-a14ccd079756.png)

**Comment by DanielnetoDotCom** [comment]
I am not sure what is wrong, check your log file for some clue

**Comment by angelawong** [comment]
I figured out what went wrong. I did not update the database. I did a new install using git and just transferred my video folder and my database from the 6.8 version. Once I did 'mysql -u root -p database < updateDb.v6.9.sql' and upgraded to 6.9, the application then showed a message that I needed to upgrade to 7.0, 7.1 and 7.2. Actually, once I got it to 7.0, the error went away. Anyway, I upgraded the database all the way to 7.2 and all is good now. So it was duh, on my part! Thanks so much! Cheers!
